apps/condo/domains/common/components/containers/GlobalStyle.tsx

```diff
@@ -152,6 +152,14 @@ export default function GlobalStyle () {
                   cursor: pointer;
                 }
               }
+              
+              .ant-input-affix-wrapper > input.ant-input, .ant-form-item-has-error .ant-input-affix-wrapper > input.ant-input {
```

Comment:
demo.dev.doma.ai and master branch already has a good looking input only  v1 has black text placeholder
Do we need these changes?